### PR TITLE
#148 Add i18n support

### PR DESCRIPTION
**Pull request by aaujon**
First try to add i18n support, translation works well but I struggle a bit adding a menu to choose the language instead of depending on currently configured locale.
I guess there is someting wrong about how I use the Looper in ui/manifest.enaml because I can't make language and theme both working, looks like the iterable variable is reused because I go a duplicate path error.
Also I'm not sure where I should react to the setting change, currently th language is selected in core/workbench.py which is I guess too early to load settings.
Ideas are welcome :)

### CODE REVIEW

File: inkcut/ui/manifest.enaml [hunk 1]
```diff
@@ -129,48 +138,55 @@ enamldef UIManifest(PluginManifest):
...
-            label = 'Logs...'
+            label = QApplication.translate("settings", "Logs")
             command = 'inkcut.ui.open_webpage'
             parameters = {'url':os.path.expanduser('~/.config/inkcut/logs/')}
         MenuItem:
             path = '/settings/theme'
-            label = 'Theme'
+            label = QApplication.translate("settings", "Theme")
+#        Looper:
+#            iterable = available_styles()+['system']
```
Comment (frmdstryr) [inline inkcut/ui/manifest.enaml]: Remove the `+ ['system']` and this works

File: inkcut/core/workbench.py [hunk 2]
```diff
@@ -155,6 +155,11 @@ def run(self):
         # Make sure ^C keeps working
         signal.signal(signal.SIGINT, signal.SIG_DFL)
 
+        locale = QtCore.QLocale.system().name()
+        qtTranslator = QtCore.QTranslator()
+        if qtTranslator.load("translations/"+locale):
+            InkcutWorkbench._instance.application._qapp.installTranslator(qtTranslator)
```
Comment (frmdstryr) [inline inkcut/core/workbench.py]: please use `self` instead of `InkcutWorkbench._instance`

File: inkcut/ui/manifest.enaml (hunk 1 above)
Comment (aaujon) [reply inkcut/ui/manifest.enaml]: it's working, I don't know how I missed that, tanks!

File: inkcut/core/workbench.py (hunk 2 above)
Comment (aaujon) [reply inkcut/core/workbench.py]: done

**Comment by aaujon** [comment]
Ok it should be pretty good now, I finally remove the menu option has I didn't manage to get it working nicely, handling it in ui plugin wasn't good because it's too late, application was already initialized and I couldn't add an option that is saved to file in core plugin. Menu option can be added later if necessary.

**Comment by frmdstryr** [comment]
Can the `translations/` folder and the path in workbench.py be moved to `inkcut/res/translations/` so it's included with the app?

**Comment by aaujon** [comment]
done and rebased on master

**Comment by frmdstryr** [comment]
Looks good, thank you.

**Pull request merged**